### PR TITLE
Add authentication

### PR DESCRIPTION
**Pull request by jvanbaarsen**
To make sure we have a secure platform, lets add some authentication.

Fixes: #4

/cc @brambokdam

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class User < ActiveRecord::Base
+  authenticates_with_sorcery!
+
+  validates :email, :password, presence: true
+  validates :email, uniqueness: true, if: -> { email.present? }
+
+  def email=(value)
+    self[:email] = value.downcase unless value.nil? unless value.nil?
```
Comment (brambokdam) [inline app/models/user.rb]: Maybe one unless is a better idea?

Comment (jvanbaarsen) [inline app/models/user.rb]: No

Comment (jvanbaarsen) [inline app/models/user.rb]: :trollface:

Comment (michiels) [inline app/models/user.rb]: Unless...

Comment (jvanbaarsen) [inline app/models/user.rb]: jsc

**Comment by brambokdam** [comment]
:shipit:

**Pull request merged**